packages/synapse-react-client/src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -213,55 +209,35 @@ export const CreateOAuthModal: React.FunctionComponent<
     }
   }
 
-  const labelSx = {
-    fontWeight: 700,
-    mb: '4px',
-  }
-
   const dangerLabelSx = {
-    ...labelSx,
     color: isEdit ? 'error.main' : undefined,
     '&.Mui-focused': {
       color: isEdit ? 'error.main' : undefined,
     },
   }
 
   const sectorIdentifierURIInput = (
-    <StyledFormControl fullWidth variant="standard" margin="normal">
-      <TextField
-        onChange={e => setSectorUri(e.target.value)}
-        placeholder="https://"
-        type="text"
-        value={sectorUri}
-        id="sectorURI"
-        label="Sector Identifier URI"
-        InputLabelProps={{ sx: dangerLabelSx }}
-      />
-    </StyledFormControl>
+    <TextField
+      onChange={e => setSectorUri(e.target.value)}
+      placeholder="https://"
+      type="text"
+      value={sectorUri}
+      id="sectorURI"
+      label="Sector Identifier URI"
+      margin="normal"
+      InputLabelProps={{ sx: dangerLabelSx }}
+      fullWidth
+    />
   )
 
   const redirectURIInputs = (
-    <StyledFormControl fullWidth variant="standard" margin="normal">
+    <>
       {redirectUris?.map((singleUri, idx) => (
         <div key={idx}>
           <TextField
             required={idx === 0}
-            InputLabelProps={{
-              sx: { ...dangerLabelSx, pointerEvents: 'auto', zIndex: 1 },
-            }}
-            label={
-              idx === 0 && (
-                <>
-                  Redirect URI(s)
-                  <Tooltip title={uriHelpMessage} placement="top">
-                    <HelpOutlineTwoTone
-                      className={`HelpButton`}
-                      sx={{ float: 'right' }}
-                    />
-                  </Tooltip>
```

Comment:
Remove tooltip as it doesn't provide substantive information and is difficult to style when included as part of the label prop